nov-13-hardfork-spec.md

```diff
@@ -25,11 +25,42 @@ To calculate the difficulty of a given block (B_n+1), with an MTP-11[1] greater
 1. Let Target (T) be equal to the (2^256 - PW) / PW.  This is calculated by taking the two’s complement of PW (-PW) and dividing it by PW (-PW / PW).
 1. The target difficulty for block B_n+1 is then equal to the lesser of T and 0x00000000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
 
+## Test Case
+
+1. Create a genesis block with the following data:
+
+```
+    nHeight = 0;
+    nTime = 1269211443;
+    nBits = 0x1c0fffff;
+```
+
+1. Add 2049 blocks at 600 second intervals with the same nBits.
+1. Add another 10 blocks at 600 second intervals.  nBits should remain constant.
+1. Add a block 6000 seconds in the future with nBits remaining the same.
+1. Add a block -4800 seconds from the previous block.  nBits should remain the constant.
+1. Add 20 blocks at 600 second intervals.  nBits should remain constant.
+1. Add a block at a 550 second interval. nBits should remain constant.
+1. Add 20 blocks at 550 second intervals. The target difficulty should slowly decrease.
```

Comment:
Presumably a typo:
It should read "Add 10 blocks at 550 second intervals" to match the test scenario.

With 20 blocks the test fails.

Reply:
Fixed. Good catch